src/Altinn.App.Core/Features/Correspondence/Models/CorrespondenceContent.cs

```diff
@@ -0,0 +1,45 @@
+using Altinn.App.Core.Models;
+
+namespace Altinn.App.Core.Features.Correspondence.Models;
+
+/// <summary>
+/// The message content in a correspondence
+/// </summary>
+public sealed record CorrespondenceContent : CorrespondenceBase, ICorrespondenceSerializer
+{
+    /// <summary>
+    /// The correspondence message title (subject)
+    /// </summary>
+    public required string Title { get; init; }
+
+    /// <summary>
+    /// The language of the correspondence, specified according to ISO 639-1
+    /// </summary>
+    public required LanguageCode<Iso6391> Language { get; init; }
+
+    /// <summary>
+    /// The summary text of the correspondence message
+    /// </summary>
+    public required string Summary { get; init; }
+
+    /// <summary>
+    /// The full text (body) of the correspondence message
+    /// </summary>
+    public required string Body { get; init; }
+
+    /// <summary>
+    /// File attachments to associate with this correspondence
+    /// </summary>
+    public IReadOnlyList<CorrespondenceAttachment>? Attachments { get; init; }
+
+    // TODO: Should this be internal?
+    /// <inheritdoc />
+    public void Serialize(MultipartFormDataContent content)
```

Comment:
Two questions on this and the other serializers @martinothamar:

- As per our naming convention, this should probably be named `Serialise`. But for whatever reason I liked the americanized spelling way more, so I stuck with it. But its not really consistent with the change I did on `UseMaskinportenAuthorisation`
- These methods are currently public, which I think makes sense. But at the same time it would be extremely strange if the correspondence models got used in a way that CorrespondenceClient wasn't involved and the caller had to do their own serialization

Reply:
Picking a different dialect of English than C# for our C# code is a great way to get discussions like this.

Reply:
Regarding your second point I also think it's strange for people to use our serialisation code direclty. Making those internal sounds good to me 👍